### PR TITLE
#167313389 Integrated HoundCi 

### PR DESCRIPTION
**Pull request by nshutijonathan**
### What does this PR do?
- Configure HoundCi
### Description of Task to be completed?
- added .hound.yml
- Integrated code-pirates backend repo with houndCi services
- added houndCii badge 
### How should this be manually tested?
- open the readme on the repo then click on the HoundCi badge available
### What are the relevant pivotal tracker stories?
[#167313389](https://www.pivotaltracker.com/n/projects/2372121/stories/167313389)
### Screenshots (if appropriate)
Screen Shot 2019-07-31 at 11 15 00

### CODE REVIEW

**Review by habinezadalvan — APPROVED** [review]
I can see that hound is actively connected to the repo.

**Review by nkalyesubula — CHANGES_REQUESTED** [review]
Nice work. Rebase though so that the file you are referencing in .hound.yml (.eslintrc)may be found

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
Comment (Mnickii) [inline .hound.yml]: Add new line

Comment (nshutijonathan) [reply .hound.yml]: > Add new line!
@Mnickii  I have added new line

**Review by salviosage — APPROVED** [review]

**Review by minega25 — APPROVED** [review]

**Comment by nshutijonathan** [comment]
> Nice work. Rebase though so that the file you are referencing in .hound.yml (.eslintrc)may be found

I have rebased

**Pull request merged**